drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodListDialogFragment.kt

```diff
@@ -43,6 +43,9 @@ class PaymentMethodListDialogFragment :
     PaymentMethodAdapter.OnPaymentMethodSelectedCallback,
     PaymentMethodAdapter.OnStoredPaymentRemovedCallback {
 
+    private var _binding: FragmentPaymentMethodsListBinding? = null
```

Comment:
We have this use case in many other places as well, we should fix the rest in another PR 👍